### ISSUE TITLE
Error in Evaluation

### ISSUE THREAD

**Issue by kansalaman** [BOUALILILila/markers_bert#2]
I am not being able to run evaluation using the following command  :
```
python ./Modeling/modeling_base.py \
      --data_dir=$DATA_DIR \
      --output_dir=$OUTPUT_DIR \
      --max_seq_length=512 \
      --do_eval \
      --per_gpu_eval_batch_size=32 \
      --per_gpu_train_batch_size=32 \
      --gradient_accumulation_steps=1 \
      --learning_rate=3e-6 \
      --weight_decay=0.01 \
      --adam_epsilon=1e-8 \
      --max_grad_norm=1.0 \
      --num_train_epochs=2 \
      --warmup_steps=10398 \
      --logging_steps=1000 \
      --save_steps=25996 \
      --seed=42 \
      --local_rank=-1 \
      --overwrite_output_dir
```
(note that I have removed do-train, since model has already been trained)

The following few are the issues I see here - 
In the code handling evaluation, `{args.data_dir}/doc2query_run/base/run_dev.csv` has been referenced. However there is no `run_dev.csv` in the first place. We downloaded `run.dev.small.tsv` in the earlier steps. Is this what is being referenced?

Secondly, if `run.dev.small.tsv` indeed is being referenced then also its in a different format (its not in the format - query, text, label). However the way the file is fed into dataset doesn't seem to handle this difference in format and hence the code fails to run.

Can someone help me out here? Quick response will be appreciated. Thanks!

**Comment by Jair-Gomes** [comment]
Did you solve this problem? I'm having a problem with other file:

 File "./Modeling/modeling_base.py", line 414, in <module> main()
  File "./Modeling/modeling_base.py", line 356, in main
    tokenizer = BertWordPieceTokenizer(f"{args.data_dir}/bert_based_uncased_vocab.txt", lowercase=True)
  File "/home/jair/.conda/envs/data_base/lib/python3.8/site-packages/tokenizers/implementations/bert_wordpiece.py", line 30, in __init__
    tokenizer = Tokenizer(WordPiece(vocab, unk_token=str(unk_token)))
Exception: Error while initializing WordPiece: No such file or directory (os error 2)

**Comment by Jair-Gomes** [comment]
Did you solve this problem?
